homeassistant/components/season/sensor.py

```diff
@@ -53,28 +57,26 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+def setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
     """Display the current season."""
-    if None in (hass.config.latitude, hass.config.longitude):
-        _LOGGER.error("Latitude or longitude not set in Home Assistant config")
-        return False
-
-    latitude = util.convert(hass.config.latitude, float)
```

Comment:
It is already a float, according to the config type hint, so there is no need to convert it.